src/validation/userValidation.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable require-jsdoc */
```

Comment:
Why did you delete the  `src/validation/emailValidation.js`, ` src/services/mail/template/verifyEmail.js` and `src/services/mail/Mail.js`?
@ola357 @conquext @OlawaleJoseph  @dewaleolaoye and @Dolapo47 is there a reason why you let it slide?

Reply:
I thought It was a conflict. I have added it back.